docs/resources/Template.md

```diff
@@ -112,3 +112,37 @@ Create a new guild based on a template. Returns a [guild](#DOCS_RESOURCES_GUILD/
 |-------|------------------------------------------|-----------------------------------------|
 | name  | string                                   | name of the guild (2-100 characters)    |
 | icon? | [image data](#DOCS_REFERENCE/image-data) | base64 128x128 image for the guild icon |
+
+## Get Guild Template % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/templates
+
+Returns an array with a [template](#DOCS_RESOURCES_TEMPLATE/template-object) object for the guild's template, or an empty array if it doesn't have one.  Requires the `MANAGE_GUILD` permission.
```

Comment:
ah, I thought that it was just to avoid breaking backward compatibility with the client from before it was changed from multiple to one, should I also change the route title to `Get Guild Templates`?

Reply:
The method should be renamed to "Get Guild Templates", too, in that case.

Reply:
Yeah let's do Get Guild Templates.